src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java

```diff
@@ -2121,6 +2118,24 @@ public void run() {
                     120);
         }
 
+        GT_Values.RA.addAssemblerRecipe(
+                new ItemStack[] { GT_OreDictUnificator.get(OrePrefixes.pipeQuadruple, Materials.Infinity, 1L),
+                        CustomItemList.Hull_UEV.get(1L), GT_Utility.getIntegratedCircuit(4) },
+                Materials.Polybenzimidazole.getMolten(2304L),
+                ItemList.Hatch_Input_Multi_2x2_UEV.get(1L),
+                600,
+                24);
+        GT_Values.RA
+                .addAssemblerRecipe(
+                        new ItemStack[] {
+                                GT_OreDictUnificator
+                                        .get(OrePrefixes.pipeQuadruple, MaterialsUEVplus.TranscendentMetal, 1L),
+                                CustomItemList.Hull_UIV.get(1L), GT_Utility.getIntegratedCircuit(4) },
+                        Materials.Polybenzimidazole.getMolten(2304L),
+                        ItemList.Hatch_Input_Multi_2x2_UIV.get(1L),
+                        600,
+                        24);
```

Comment:
Why these recipes don't use RA2?

Reply:
the file is massive and the author didnt convert everything yet (nor did they claim to). those 2 just moved